configuration/filters.md

```diff
@@ -0,0 +1,90 @@
+---
+description: >-
+  This document outlines the various Filters types supported by the Edge Delta
+  agent, and how the filters are configured for inputs.
+---
+
+# Filters
+
+## Overview
+
+Filters are the mechanism to filter and transform the received logs from input sources.
+
+Proper filter configuration reduces resource usage, increases performance and hides or masks confidential data. It also provides a way to group the related events before further processing.
```

Comment:
how does filters provide a way to group events? are you thinking of buffered trace processor filter? maybe we don't have to mention about that grouping in the overview section.

Reply:
Yes it is buffered tracer.  So I guess we won't plan to mentioned it in the public docs. I am removing that remark.

Reply:
I will add it to this doc once you merge. but mention about it at the bottom in its own section to leave Overview simple.